contrib/datastore/file/gcp/gcp.go

```diff
@@ -85,27 +94,232 @@ func (g GcpAdapter) CreateFileShare(ctx context.Context, fs *file.CreateFileShar
 }
 
 func (g GcpAdapter) GetFileShare(ctx context.Context, fs *file.GetFileShareRequest) (*file.GetFileShareResponse, error) {
-	panic("implement me")
+	isCreateOpDone, err := g.isOperationDone(createOp)
```

Comment:
Why do we need to check the completion of all these operations for getting fileshare info? Please put appropriate comment

Reply:
Sync DB require to update the resource metadata using GET call from cloud backend. That's why these check are required here.